onnxruntime/python/tools/transformers/microbench/benchmark.py

```diff
@@ -0,0 +1,30 @@
+import time
+import onnxruntime as ort
+
+
+def create_session(onnx_file, provider, profiling=False):
+    sess_opt = ort.SessionOptions()
+    sess_opt.enable_profiling = profiling
+    if provider == "rocm":
```

Comment:
should we make it more flexible for other EPs?

Reply:
Yes, but maybe later. In general, I agree with all the comments; we can definitely extend this utility incrementally as the needs grow.